.gitignore

```diff
@@ -25,11 +18,14 @@ override.tf.json
 *_override.tf.json
 
 # Include override files you do wish to add to version control using negated pattern
-# !example_override.tf
+!tests_override.tf
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
 
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+
+# Ignore rc files
+.envrc
```

Comment:
Also make a change to upstream [module template](https://github.com/cloudeteer/terraform-module-template/blob/main/.gitignore) for this.